tests/ui/test_pvc_ui.py

```diff
@@ -20,25 +21,43 @@ def teardown(self):
         pvcs = [pvc_obj for pvc_obj in pvc_objs if "test-pvc" in pvc_obj.name]
         delete_pvcs(pvc_objs=pvcs)
 
-    @ui
+    @tier1
```

Comment:
are we planning to run this as tier1 because there is a marker change

Reply:
yes, it was discussed in the last meeting. @ebenahar @OdedViner  Please confirm.

Reply:
yes